experimental/PropertyDDS/packages/property-shared-tree-interop/package.json

```diff
@@ -52,6 +52,18 @@
 		"ts-jest": "^29.1.1",
 		"typescript": "~5.1.6"
 	},
+	"fluidBuild": {
+		"tasks": {
+			"eslint": [
+				"build:esnext",
+				"build:test"
+			],
+			"build:test": [
+				"build:esnext",
+				"^tsc"
+			]
+		}
+	},
```

Comment:
If I got this right, esm-only packages need special handling here because the default task dependency graph only works over the CJS build (`commonjs`)? Feels like something we might want to address there at some point.

Reply:
The first step in doing that is to rename the tsc task and stop it from being special: https://github.com/microsoft/FluidFramework/pull/19039